code/components/gta-core-five/src/PatchAttachCD2545.cpp

```diff
@@ -1,32 +1,54 @@
 #include <StdInc.h>
 #include <Hooking.h>
 #include "CrossBuildRuntime.h"
+#include <ScriptEngine.h>
+#include "ICoreGameInit.h"
 
 //
-// This patch forces a tunable (seen starting at b2545) preventing applying
-// of ped-to-ped attachment data to networked peds to be 'off', returning the
-// old behavior for compatibility.
+// This patches multiple checks that prevent the application of entity-to-ped
+// attachment data to networked peds so it can be dynamically toggled.
 //
 
 static bool ReturnFalse()
 {
 	return false;
 }
 
+static bool g_disableRemotePedAttachment = false;
+
+static bool ReturnState()
+{
+	return g_disableRemotePedAttachment;
+}
+
 static HookFunction hookFunction([]()
 {
 	if (xbr::IsGameBuildOrGreater<2545>())
 	{
-		auto location = hook::get_pattern<char>("BA E7 8F A5 1E B9 BD C5 AF E3 E8");
-		hook::call(location + 23, ReturnFalse);
+		const auto location = hook::get_pattern<char>("BA E7 8F A5 1E B9 BD C5 AF E3 E8");
+		hook::call(location + 23, ReturnState);
 
 		// This call was removed in 2802.0
 		if (!xbr::IsGameBuildOrGreater<2802>())
 		{
 			hook::call(location + 61, ReturnFalse);
 		}
 
-		auto attachEntityToEntityTunable = hook::get_pattern<char>("BA 37 89 3D 8A B9 BD C5 AF E3");
+		const auto attachEntityToEntityTunable = hook::get_pattern<char>("BA 37 89 3D 8A B9 BD C5 AF E3");
 		hook::call(attachEntityToEntityTunable + 23, ReturnFalse);
 	}
 });
+
+static InitFunction initFunction([]()
+{
+	fx::ScriptEngine::RegisterNativeHandler("ONESYNC_ENABLE_REMOTE_ATTACHMENT_SANITIZATION", [](fx::ScriptContext& context)
```

Comment:
What do you think if instead of adding new native function for each such flag we just add
SET_CFX_CONFIG_FLAG, GET_CFX_CONFIG_FLAG and RESET_CFX_CONFIG_FLAG (will reset to default) and there will be a service that will control such flags. We already have many of this, like FLY_THROUGH_WINDSHIELD, ragdoll on R and so on.
If you can't/don't want to do it - I will do it anyway, but can't say when

Reply:
Sounds good! Just to make sure: Are we still handling values internally with a ConVar then?

Like your example:
We keep `game_enableFlyThroughWindscreen` as a ConVar and allow interaction with the new exposed Setters/Getters? My only question then would be how we'd handle existing behavior for flags like this one, as they're currently treated as replicated ConVars which would allow no client-side set operation.

Will take a look!

Reply:
At least I want to see it consistent. Some of setters we have on server side, some on client. Let's add everything on server side and replicate if it already works like this